tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "declaration": true,
     "forceConsistentCasingInFileNames": true,
     "inlineSources": true,
-    "jsx": "react",
+    "jsx": "react-jsx",
```

Comment:
Now that we support the `exports` field in `package.json`, we could have a prod bundles using `react-jsx`, and dev bundles using `react-jsxdev`. 🤔